src/Messages/Interactions/TSIncomingMessage.h

```diff
@@ -120,6 +104,7 @@ extern NSString *const TSIncomingMessageWasReadOnThisDeviceNotification;
 + (nullable instancetype)findMessageWithAuthorId:(NSString *)authorId timestamp:(uint64_t)timestamp;
 
 @property (nonatomic, readonly) NSString *authorId;
+@property (nonatomic, readonly) UInt32 sourceDeviceId;
```

Comment:
Yup, should be fine. 

Though it probably warrants at least hazarding comment about how this value will be 0 for older messages in case someone wants to repurposes this field later. I'll add one.